simde/x86/sse4.2.h

```diff
@@ -37,6 +37,231 @@ SIMDE__BEGIN_DECLS
 #  define SIMDE_X86_SSE4_2_ENABLE_NATIVE_ALIASES
 #endif
 
+#define SIMDE_SIDD_CMP_EQUAL_ANY     0
+#define SIMDE_SIDD_CMP_RANGES     1
+#define SIMDE_SIDD_CMP_EQUAL_EACH     2
+#define SIMDE_SIDD_CMP_EQUAL_ORDERED   3
+
+#if defined(SIMDE_X86_SSE4_2_ENABLE_NATIVE_ALIASES)
+#define _SIDD_CMP_EQUAL_ANY SIMDE_SIDD_CMP_EQUAL_ANY
+#define _SIDD_CMP_RANGES SIMDE_SIDD_CMP_RANGES
+#define _SIDD_CMP_EQUAL_EACH SIMDE_SIDD_CMP_EQUAL_EACH
+#define _SIDD_CMP_EQUAL_ORDERED SIMDE_SIDD_CMP_EQUAL_ORDERED
+#endif
+
+SIMDE__FUNCTION_ATTRIBUTES
+int
+simde_mm_cmpestra_8_(simde__m128i a, int la, simde__m128i b, int lb, const int imm8) {
```

Comment:
All compilers require a compile-time constant here (see https://godbolt.org/z/8gBpfe), so you should add a `SIMDE_REQUIRE_CONSTANT(imm8)`.  Note that this may require you to change `simde_mm_cmpestra` to be a macro; that's okay, something like

```c
#define simde_mm_cmpestra(a, la, b, lb, imm8) \
  (((imm8) & SIMDE_SIDD_UWORD_OPS) \
    ? simde_mm_cmpestra_16_((a), (la), (b), (lb), (imm8)) \
    : simde_mm_cmpestra_8_((a), (la), (b), (lb), (imm8)))
```

should be fine.

Reply:
changed it too, I hope it is fine

Reply:
@masterchef2209 I'm not seeing your changes about https://github.com/nemequ/simde/pull/280/files#r423221728 ?

Reply:
I have changed it now, you can find it on line 288 in this file, if I have done something wrong please inform me.
Thank You